app/_locales/en/messages.json

```diff
@@ -3617,6 +3617,12 @@
   "personalAddressDetected": {
     "message": "Personal address detected. Input the token contract address."
   },
+  "petnamesEnabledToggle": {
+    "message": "Petnames"
+  },
+  "petnamesEnabledToggleDescription": {
+    "message": "This allows you to assign a nickname to any address. This nickname will be visible to you only."
```

Comment:
Worth reviewing via PM also.

I'd vote not to use the term `nickname` as `petname` is itself the technical term. Maybe just `name` and likely worth mentioning it will suggest names also.

Reply:
cc @bschorchit can you take a look?

Reply:
@coreyjanssen would you mind reviewing this copy the for Petnames toggle within settings?
`"This allows you to assign a nickname to any address. This nickname will be visible to you only."`

One thing I think we should also mention is that it also _allows you to view proposed names for addresses that you're interacting with when those are available_ (not in these exact words, obviously 😅)

You can find more details on the petname feature here: https://www.figma.com/file/t0JK0HxVVlDkP7lQ9hyYag/Petnames?type=design&node-id=11-12784&mode=design&t=EAczA19dL9m5mZXh-0

Reply:
Combined the copy for both requests here for a more cohesive content flow in the settings section. @bschorchit wdyt?

`This lets you assign a name to any address. We'll suggest names for addresses that you interact with when possible. We get this data from third-party sources like Etherscan, Infura, and Lens Protocol. These sources will be able to see your IP address if you turn on this feature. `